### PR TITLE
Port IVT to Microsoft.VisualStudio.PortableLibrary added in VSO copy …

### PR DESCRIPTION
**Pull request by mavasani**
…of the Microsoft.VisualStudio.Editors project

### CODE REVIEW

**Comment by mavasani** [comment]
Tagging @dotnet/project-system for review

**Comment by mavasani** [comment]
Also tagging @shyamnamboodiripad

**Comment by mavasani** [comment]
Fixes VSO [244923](https://devdiv.visualstudio.com/0bdbc590-a062-4c3f-b0f6-9383f67865ee/_workitems?id=244923)

**Comment by shyamnamboodiripad** [comment]
:+1:

File: src/Microsoft.VisualStudio.Editors/My Project/AssemblyInfo.vb [hunk 1]
```diff
@@ -38,6 +38,13 @@ Imports System.Runtime.InteropServices
  "c1ab99285d622caa652c1dfad63d745d6f2de5f17e5eaf0fc4963d261c8a12436518206dc09334" & _
  "4d5ad293")>
 
+<Assembly: InternalsVisibleTo("Microsoft.VisualStudio.PortableLibrary, PublicKey=" & _
```
Comment (mavasani) [inline src/Microsoft.VisualStudio.Editors/My Project/AssemblyInfo.vb]: I also confirmed that the IVTs are up-to-date for the AppDesigners project: [VSO](https://devdiv.visualstudio.com/DevDiv/_git/VS?path=%2Fsrc%2Fwizard%2Fvbdesigner%2FAppDesigner%2FMy%20Project%2FAssemblyInfo.vb&version=GBmaster&_a=contents) and [GitHub](https://github.com/dotnet/roslyn-project-system/blob/master/src/Microsoft.VisualStudio.AppDesigner/My%20Project/AssemblyInfo.vb)

**Comment by srivatsn** [comment]
Why was this missing? Was this added recently?

**Comment by mavasani** [comment]
Yes - see [commit](https://devdiv.visualstudio.com/DevDiv/_git/VS/commit/6cf1cacc854c7726ee4930f1c22761dd6d73b9e8?path=%2Fsrc%2Fwizard%2Fvbdesigner%2Fdesigner%2FMy%20Project%2FAssemblyInfo.vb&_a=compare)

**Comment by srivatsn** [comment]
👍

**Pull request merged**